modeshape-persistence-tests/pom.xml

```diff
@@ -0,0 +1,417 @@
+                                <id>integration-test-mysql</id>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                </goals>
+                                <configuration>
+                                    <systemPropertyVariables>
+                                        <!-- Make these available to the tests via system properties -->
+                                        <database.driver>com.mysql.jdbc.Driver</database.driver>
+                                        <database.url>jdbc:mysql://${database.hostname}:3306/${database.name}</database.url>
+                                    </systemPropertyVariables>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>verify</id>
+                                <goals>
+                                    <goal>verify</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        
+        <profile>
+            <id>postgres</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>integration-test-postgres</id>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                </goals>
+                                <configuration>
+                                    <systemPropertyVariables>
+                                        <!-- Make these available to the tests via system properties -->
+                                        <database.driver>org.postgresql.Driver</database.driver>
+                                        <database.url>jdbc:postgresql://${database.hostname}/${database.name}</database.url>
+                                    </systemPropertyVariables>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>verify</id>
+                                <goals>
+                                    <goal>verify</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <profile>
+            <id>assembly</id>
```

Comment:
IMO the assembly profile should not be here. Moreover, this module should not be run neither as part of `assembly` nor `disitribution`. This module should me more akin to `performance`